scripts/android-apk.bash

```diff
@@ -52,14 +52,14 @@ echo "API: $ANDROIDAPI"
 # export ANDROID_TARGET_ARCH=${ARCH}
 
 mkdir -p ${BUILD_DIR}/.gradle
-# androiddeployqt needs gradle and downloads it to /root/.gradle. By linking it to the build folder, this will be cached between builds.
-ln -s ${BUILD_DIR}/.gradle /root/.gradle
 
 pushd ${BUILD_DIR}
-cp ${SOURCE_DIR}/scripts/ci/config.pri ${SOURCE_DIR}/app/config.pri
+
 ${QT_BASE}/bin/qmake -spec android-clang ANDROID_ABIS="${ARCH}" ${SOURCE_DIR}/app/input.pro
+ls ${ANDROID_NDK_ROOT}/prebuilt/
 ${ANDROID_NDK_ROOT}/prebuilt/${ANDROID_NDK_HOST}/bin/make qmake_all
 make -j ${CORES}
+
```

Comment:
wondering if it is not worth to have android-apk.bash and android-aab.bash  as one script, or even do not have bash script and have it as part of yml file... because you build aab and both apks same way., no?